changelog/670.feature.rst

```diff
@@ -0,0 +1,9 @@
+|commands| Add support for injected parameters autocompletion
+- Document :class:`.ext.commands.Injection`
+- :meth:`Injection.register <ext.commands.Injection.register>` now returns :class:`Injection <ext.commands.Injection>`
+- :func:`.ext.commands.register_injection` now returns :class:`.ext.commands.Injection`
+- Add decorator interface for :func:`.ext.commands.inject`: :func:`.ext.commands.injection`
+- Add :meth:`.ext.commands.Injection.autocomplete` decorator
+- Add ``autocompleters`` keyword-only argument to :class:`.ext.commands.Injection` constructor
+- Add ``autocompleters`` keyword-only argument to :func:`.ext.commands.inject`
+- Add ``autocompleters`` keyword-only argument to :func:`.ext.commands.register_injection` constructor
```

Comment:
```suggestion
|commands| Add support for injected parameters autocompletion
- Add :meth:`Injection.autocomplete <ext.commands.Injection.autocomplete>` decorator
- :func:`register_injection <ext.commands.register_injection>` now returns an :class:`Injection <ext.commands.Injection>` object
- Add :func:`injection <ext.commands.injection>` as a decorator interface for :func:`inject <ext.commands.inject>`
- Add ``autocompleters`` keyword-only argument to :class:`Injection <ext.commands.Injection>`, :func:`inject <ext.commands.inject>`, and :func:`register_injection <ext.commands.register_injection>`
```
couple minor changes:
- removed `ext.commands` prefix - the raw rst is a bit more cumbersome to write, but the end result is easier to read
- removed line about `Injection.register`, not documented as per previous conversation
- merged last three lines into one

The part about `Injection` now being documented should ideally be in a separate `670.doc.rst` changelog entry.